.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - run:
           name: setup linux
           command: |
-            IMAGEMAGICK_VERSION=6.9.10-35 bash before_install_linux.sh
+            IMAGEMAGICK_VERSION=6.9.10-38 bash before_install_linux.sh
```

Comment:
This should be changed to 39 that has just been released.

Reply:
They are not yet available on https://ftp.icm.edu.pl/pub/graphics/ImageMagick/binaries/